src/crypto/ssh/bcrypt_pbkdf.cpp

```diff
@@ -82,7 +82,7 @@ bcrypt_hash(const quint8* sha2pass, const quint8* sha2salt, quint8* out)
         cdata[i] = Blowfish_stream2word(ciphertext, sizeof(ciphertext),
             &j);
     for (i = 0; i < 64; i++)
-        blf_enc(&state, cdata, sizeof(cdata) / sizeof(uint64_t));
+        blf_enc(&state, cdata, BCRYPT_WORDS / 2);
```

Comment:
Any reason to make this less explicit? The result is the same.

Reply:
Clang 10 marks it as a warning and thus won't compile

Reply:
Which warning is that? Suspicious sizeof / sizeof? What about changing `sizeof(uint64_t)` to `sizeof(uint32_t) / 2`? or `8` or something?

Reply:
I generally hate determining array length with sizeof. In this case it makes little sense to do that anyway, the variable the functions asks for is "blocks" which are sequential pairs in the array.